packages/nimble-components/src/table/components/cell/styles.ts

```diff
@@ -32,5 +33,8 @@ export const styles = css`
         margin-left: auto;
         flex-shrink: 0;
         flex-grow: 0;
+        width: ${controlSlimHeight};
+        height: ${controlSlimHeight};
```

Comment:
Ah dang it, srry molly, just remembered. Can you use this aspect-ratio pattern so we can do this more consistently (also makes it less weird to assign a height to a width)?

https://github.com/ni/nimble/blob/97d4deddb38093b51ca3916aac6cf4ccab138415/packages/nimble-components/src/spinner/styles.ts#L16-L17

Reply:
Good idea 👍

Reply:
hmm...it seems like that change caused a styling change in safari on all of our icon buttons.

Reply:
After some investigation, I discovered that Chromatic uses Safari 16.0 (released September 2022), and the icon button renders as 34x32 in that version of Safari rather than 32x32 as it does in the latest version of Safari.

Should we come up with a solution that renders correctly in Safari 16.0, such as not using the `aspect-ratio` styling? Or, should it be a "known issue" that the Chromatic snapshots of icon buttons are not the correct dimensions?

Reply:
As additional input to the decision, [Chromatic has historically updated browser versions very infrequently](https://www.chromatic.com/docs/infrastructure-release-notes), roughly 1-2 times per year. The last release was February 2023 so if we opt for a "workaround" solution that's not our preferred long-term practice then it might need to remain in place for quite a while.

I waffled quite a bit but I think I'm ok with accepting the Safari diffs and calling this a "known issue" since the version that exhibits the bug is outside our 6 month support window. (Others can feel free to push for a different approach). 

If we go that direction, I can't think of a great way to track the known issue. [Chromatic auto-accepts diffs when they do an infrastructure update](https://www.chromatic.com/docs/browsers?ref=chromaticblog.ghost.io#browser-upgrades) meaning there won't be any action for us to take when it's fixed, so probably not much value in filing a bug. I vote we just accept the diffs and move on.